Application/DependencyInjection.cs

```diff
@@ -18,6 +19,7 @@ public static class DependencyInjection
         public static void AddApplication(this IServiceCollection services)
         {
             services.AddAutoMapper(cfg => cfg.AddMaps(typeof(WorkplacesProfile).Assembly));
+            services.AddAutoMapper(cfg => cfg.AddMaps(typeof(BookingProfile).Assembly));
```

Comment:
not needed, 1 row is enough

Reply:
What do you mean ? Adding all the assemlies at once ?